### DISCUSSION TITLE
Generate folder thumbnail with the first image file in the folder

### DISCUSSION THREAD

**Discussion by poyichou** [9001/copyparty#36] [Ideas]
Is it possible to take an image as a folder thumbnail if there is no folder.png, folder.jpg, cover.png, cover.jpg in the folder?
I think the easiset way is to sort images in a folder by name, and take the 1st one as the thumbnail.
I know I can make a thumbnail by copying an image and name it to cover.jpg, but I don't want to change the date of the folder.
Thanks.

**Comment by 9001** [comment]
Good news -- this feature was added in version 1.6.8 :-)

The problem is that, due to performance reasons, [file indexing](https://github.com/9001/copyparty#file-indexing) must be enabled for this to work. So please try to enable that, either globally with `-e2dsa`, or just for one volume with the `e2ds` volflag, and see if that helps.

Thank you for asking btw! You made me realize I forgot to document this behavior, so I'll fix that before the next version 👍 

And thx for using copyparty :-)

**Comment by poyichou** [comment]
Thx for the speedy reply :)
I successfully enabled this feature by specifying -e2dsa with command.
